backend/coreapp/serializers.py

```diff
@@ -146,23 +150,38 @@ class Meta:
             "platform",
         ]
 
-    def get_project(self, scratch: Scratch):
+    # TODO: Unused
+    def get_project(self, scratch: Scratch) -> Optional[str]:
```

Comment:
@nanaian do you know why these are here? I don't think we need them on the serializer, and they seem unused

Reply:
these are not unused
https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield

Reply:
okay, thanks. They used to not always return a value (the function would fall through to nothingness) - is the `return None` I added at the end of them (and the return type `str`) correct?

Reply:
yep LGTM